query-sql/src/lib.rs

```diff
@@ -280,8 +305,15 @@ impl QueryFragment for FromClause {
     fn push_sql(&self, out: &mut QueryBuilder) -> BuildQueryResult {
         use self::FromClause::*;
         match self {
-            &TableList(ref table_list) => table_list.push_sql(out),
-            &Join(ref join) => join.push_sql(out),
+            &TableList(ref table_list) => {
+                out.push_sql(" FROM ");
```

Comment:
You don't have to do this now, but further to my trait comment earlier, you could implement `" FROM ".push_sql(out)`, both for symmetry and potentially for easier use with `itertools`, etc.